### ISSUE TITLE
async branch is not compatible with python 3.7

### ISSUE THREAD

**Issue by cbergmiller** [riptideio/pymodbus#320]
`async` is a not allowed in Python 3.7 module paths.

### Versions

* Python: 3.7
* OS: Windows
* Pymodbus: async branch

### Description

The async branch can not be used with Python 3.7
A solution would be to rename `pymodbus.client.async` with `pymodbus.client.aio` or `pymodbus.client.asynchronous`.

I can create a PR if desired.

### Code and Logs

```python
  File "async_asyncio_client.py", line 18
    from pymodbus.client.async.tcp import AsyncModbusTCPClient as ModbusClient
                             ^
SyntaxError: invalid syntax
```

**Comment by dhoomakethu** [comment]
@cbergmiller thanks for bringing this to notice, do you have any more hints on why this is happening ? Is `async` a reserved keyword starting with 3.7 ?

**Comment by cbergmiller** [comment]
@dhoomakethu yes `async` and `await` are reserved keywords since 3.7:
[https://docs.python.org/3/whatsnew/3.7.html](https://docs.python.org/3/whatsnew/3.7.html)

**Comment by dhoomakethu** [comment]
@cbergmiller Appreciate if you can raise a PR . Thanks.

**Comment by cbergmiller** [comment]
It would be great if Python 3.7 could be added to the Travis CI test environments.

**Comment by jessekerkhoven** [comment]
Any update on merging `dev3.7` branch in to master for a new release? See also #346

**Comment by dhoomakethu** [comment]
@jessekerkhoven apologies for the delay, I am yet to do a code review . Will try to complete this asap.

**Comment by dhoomakethu** [comment]
For any one wanting support on python 3.7, please give a try on branch [pymodbus-2.2.0](https://github.com/riptideio/pymodbus/tree/pymodbus-2.2.0)

**Comment by dhoomakethu** [comment]
Fixed and available as part of [v2.2.0rc1](https://github.com/riptideio/pymodbus/releases/tag/v2.2.0rc1).
`pip install pymodbus==2.2.0rc1`
